src/object.d

```diff
@@ -3190,11 +3199,24 @@ void __ctfeWriteln(T...)(auto ref T values) { __ctfeWrite(values, "\n"); }
  * Create RTInfo for type T
  */
 
+template RTInfoImpl(size_t[] pointers)
+{
+    immutable size_t[pointers.length] data = pointers[];
+    immutable RTInfoImpl = data.ptr;
```

Comment:
Might also make sense to inline the bits for small block pools, e.g.
`[M bytes of pointer bitmap, e.g. 16 for alignment] + (M * 8 / N) x [N bytes block]`.
Can we come up with a good scheme for that?